### ISSUE TITLE
Check deploy and fix deploy from new repository location on github

### ISSUE THREAD

**Issue by sfisher** [CDL-Dryad/dryad-product-roadmap#21]
Ryan is moving our current repositories over and I will need to get the deploy working again.

**Comment by sfisher** [comment]
I may also need to change or add travis.ci to get automated testing from the new repositories.

**Comment by sfisher** [comment]
I believe the capistrano deploy may work but I will need to wait to get the new development server set up before I can really test.

**Comment by briri** [comment]
The deploy script is working now. Calls to https://dryad-dev.cdlib.org now resolve to passenger.

There seems to be an issue though with some of the config settings though. The following line is resolving to `none_login_path`.
https://github.com/CDL-Dryad/stash/blob/a711e0dc2eaa433df32f41f2d8c9b8f0b13e86fd/stash_engine/app/models/stash_engine/tenant.rb#L47

**Comment by marisastrong** [comment]
Yes we will be using shib for the dev installation.

**Comment by sfisher** [comment]
Yes.  I think the deploy itself works, but the shibboleth doesn't work for login yet.  For now choose hte option for "Not Dryad partner" until the shibboleth stuff is set up and working.
